great_expectations/render/renderer/site_builder.py

```diff
@@ -402,6 +405,17 @@ def add_resource_info_to_index_links_dict(self,
     
         return index_links_dict
 
+    #TODO: Fill out this method to make cta footer dynamic
```

Comment:
This is the method that generates the cta_object that gets passed to the index page renderer.